### DISCUSSION TITLE
2005/12/11/%E6%AF%94%E8%BE%83%E9%AB%98%E6%98%8E%E7%9A%84%E9%AA%97%E6%9C%AF

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#68] [Announcements]
# 2005/12/11/%E6%AF%94%E8%BE%83%E9%AB%98%E6%98%8E%E7%9A%84%E9%AA%97%E6%9C%AF

洋骗子里还是有些高明的人的，刚看到一个，转过来：

https://blog.qizhen.xyz/2005/12/11/%E6%AF%94%E8%BE%83%E9%AB%98%E6%98%8E%E7%9A%84%E9%AA%97%E6%9C%AF

**Comment by ruanqizhen** [comment]
之前博客下的留言：


少华
2007年05月26日 22:20 编辑
并不高明，但做法确实很独到啊

回复
fish
2007年04月19日 13:59 编辑
真是高啊，以前国内也有人做过的预测。真是害人不浅

回复
Zhen
2005年12月12日 16:01 编辑
强！佩服佩服

回复
奕瑋
2005年12月12日 10:07 编辑
高，实在是高！

回复
